packages/core/content-releases/admin/src/components/tests/CMReleasesInjectionZone.test.tsx

```diff
@@ -0,0 +1,113 @@
+import { useCMEditViewDataManager } from '@strapi/helper-plugin';
+import { screen } from '@testing-library/react';
+import { render, server } from '@tests/utils';
+import { rest } from 'msw';
+
+import { CMReleasesInectionZone } from '../CMReleasesInjectionZone';
+
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  // eslint-disable-next-line
+  CheckPermissions: ({ children }: { children: JSX.Element }) => <div>{children}</div>,
```

Comment:
ℹ️ I see this is what we do throughout the codebase, but shouldn't test the actual based on given permissions 🤔 ?